packages/components/src/navigator/navigator-screen/component.tsx

```diff
@@ -31,22 +26,11 @@ import { contextConnect, useContextSystem } from '../../context';
 import { useCx } from '../../utils/hooks/use-cx';
 import { View } from '../../view';
 import { NavigatorContext } from '../context';
+import * as styles from '../styles';
 import type { NavigatorScreenProps } from '../types';
 
-const animationEnterDelay = 0;
-const animationEnterDuration = 0.14;
-const animationExitDuration = 0.14;
-const animationExitDelay = 0;
-
-// Props specific to `framer-motion` can't be currently passed to `NavigatorScreen`,
-// as some of them would overlap with HTML props (e.g. `onAnimationStart`, ...)
-type Props = Omit<
-	WordPressComponentProps< NavigatorScreenProps, 'div', false >,
-	Exclude< keyof MotionProps, 'style' | 'children' >
->;
-
 function UnconnectedNavigatorScreen(
-	props: Props,
+	props: WordPressComponentProps< NavigatorScreenProps, 'div', false >,
```

Comment:
Nice simplification of the types 👍 

Any backward compatibility concerns? For example, we won't be able to use any of the framer motion animation events like `onAnimationStart` / `onAnimationComplete`.

Reply:
That shouldn't be a problem, because `Navigator` was previously _not_ accepting props specific to `framer-motion` that would overlap with standard HTML properties (like `onAnimationStart`).

So, with the changes from this PR, the component should now accept the default HTML `onAnimationStart`